assets/js/components/coverage/CoverageHotspotShow.jsx

```diff
@@ -0,0 +1,153 @@
+  const hotspotAlias = orgHotspot ? orgHotspot.alias : null
+
+  return (
+    <div style={{ padding: 25, paddingTop: 8 }}>
+      <Button
+        icon={<ArrowLeftOutlined style={{ fontSize: 12 }} />}
+        style={{ border: "none", padding: 0, fontSize: 14, color: '#bfbfbf', height: 24 }}
+        onClick={() => props.selectHotspotAddress(null)}
+      >
+        Back
+      </Button>
+
+      <Row gutter={12}>
+        <Col sm={20}>
+          <div style={{ marginBottom: 8, display: 'flex', flexDirection: 'row', alignItems: 'center' }}>
+            <Text style={{ fontSize: 22, fontWeight: 600, marginRight: 20 }}>{startCase(hotspot.hotspot_name)}</Text>
+            {renderStatusLabel(hotspot.status)}
+          </div>
+          <div style={{ display: 'flex', flexDirection: 'row', alignItems: 'center', marginBottom: 12 }}>
+            <span style={{ color: '#2C79EE', marginRight: 20, display: 'flex', flexDirection: 'row', alignItems: 'center' }}>
+              <img
+                draggable="false"
+                src={LocationIcon}
+                style={{ height: 12, marginRight: 4 }}
+              />
+              {locationString(hotspot.long_city, hotspot.short_state, hotspot.short_country)}
+            </span>
+            {
+              showAliasInput ? (
+                <OutsideClick
+                  onOutsideClick={() => {
+                    toggleAliasInput(false)
+                    updateHotspotAlias(hotspot.hotspot_address, name)
+                  }}
+                >
+                  <Input
+                    name="name"
+                    value={name}
+                    onChange={(e) => setName(e.target.value)}
+                    style={{
+                      width: 200,
+                      verticalAlign: "middle",
+                    }}
+                    suffix={`${name.length}/50`}
+                    maxLength={50}
+                  />
+                </OutsideClick>
+              ) : (
+                <Link
+                  to="#"
+                  onClick={e => {
+                    e.preventDefault()
+                    toggleAliasInput(true)
+                    setName(hotspotAlias || "")
+                  }}
+                >
+                  <span style={{ color: '#2C79EE', marginRight: 20, display: 'flex', flexDirection: 'row', alignItems: 'center' }}>
+                    <img
+                      draggable="false"
+                      src={AliasIcon}
+                      style={{ height: 12, marginRight: 4 }}
+                    />
+                    <Tooltip title="Click to Edit Alias">
+                      {hotspotAlias || "No Alias"}
+                    </Tooltip>
+                  </span>
+                </Link>
+              )
+            }
+          </div>
+          <div style={{ marginBottom: 8 }}>
+            <Text style={{ fontSize: 15 }}>
+              This Hotspot has heard from of your devices over the past 30 days.
```

Comment:
there should be a number of devices here, right? and an alternative msg (`none of`) for when there's no org packets heard by this hotspot?

Reply:
also are we changing this to 2 days since i think that's how we're calculating stats?

Reply:
will add the device stats next with the graph stuff in a separate PR.